tutorials/plantUml.md

```diff
@@ -278,3 +278,51 @@ package "Rule Of Thumb";{
 
 Explicitly define all symbols to avoid any potential layout mishaps.
 </box>
+
+### Using reference frames
+
+<table>
+<colgroup>
+<col style="width: 70%" />
+<col style="width: 30%" />
+</colgroup>
+
+<thead>
+<tr class="header">
+  <th>Source</th>
+  <th>Result</th>
+</tr>
+</thead>
+
+<tbody>
+
+<tr class="odd">
+<td>
+<pre>
+ref over MSLogic, TextUi : get minefield appearance
```

Comment:
A comment would be helping to tell students that this is for the reference frame only.
OR
Moreover, it might be better to simplify the image. It could focus on the section between `:TextUi` and `:MsLogic`. Then, include the entire code within the Source block, similar to how the full code for the overall image in above sections

Reply:
Thank you for the suggestion!

I have went ahead and simplified the diagrams as much I could. I have not simplified it fully as I was concerned that it could undermine the example's relevance. If the diagrams are too simplified, it might be harder to see and connect the usage of reference frames in an actual use case.

Could you suggest any specific areas that could be removed, or do you think the current diagrams are sufficient?